docs/services/qix-engine/script_reference/counter_aggregation_functions.md

```diff
@@ -1,4 +1,4 @@
-# Counter Aggregation functions
+# Counter aggregation functions
 
```

Comment:
Isn't 'A' should be capitalized for better view in aggregation?

Reply:
I will update the PR